mlir/lib/Dialect/OpenACC/IR/OpenACC.cpp

```diff
@@ -435,7 +435,7 @@ static LogicalResult verifyInitLikeSingleArgRegion(
 LogicalResult acc::PrivateRecipeOp::verifyRegions() {
   if (failed(verifyInitLikeSingleArgRegion(*this, getInitRegion(),
                                            "privatization", "init", getType(),
-                                           /*verifyYield=*/true)))
+                                           /*verifyYield=*/false)))
```

Comment:
nit: maybe we can use `hlfir::getFortranElementOrSequenceType` in the verifier to check that the sequence/scalar type of the recipe and the yield match vs just dropping verification completely.

Reply:
Unfortunately, the verifier is in the core MLIR and there is no notion of FIR type here. Maybe the element type could be retrieve through an interface somehow.

Reply:
Right, it looks like a type interface that can be attached to FIR types might be useful here.  I am okay with having verification disabled for the time being.